packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.less

```diff
@@ -3,6 +3,7 @@
 .telecom-telephony-line-phone-order {
   .phone-item {
     margin: 8px 0;
+    height: 300px;
```

Comment:
why 300px? why not 100%? let it occupy the height it needs.

Reply:
100% doesn't fix the issue some tiles don't have the same content, so they're smaller.

Reply:
Probably `min-height` could be safer here but the best option we could figure out is that replacing actual grid by using flex.
Could be handle into a separate Pull Request.